browser/about_flags.cc

```diff
@@ -111,6 +113,10 @@ constexpr char kBraveDomainBlockName[] = "Enable domain blocking";
 constexpr char kBraveDomainBlockDescription[] =
     "Enable support for blocking domains with an interstitial page";
 
+constexpr char kBraveDebounceName[] = "Enable debouncing";
+constexpr char kBraveDebounceDescription[] =
+    "Enable support for skipping tracking URLs";
```

Comment:
@pes10k should we say something like top-level redirect tracking urls? Otherwise I think it might be confusing

Reply:
yea, i think thats a good change

Reply:
Fixed in https://github.com/brave/brave-core/commit/d7282071de52bd882fd8a55d06efeef3e99795f9